nucliadb/src/nucliadb/ingest/orm/brain.py

```diff
@@ -483,6 +515,8 @@ def process_field_metadata(
         relation_node_document: RelationNode,
         user_canceled_labels: set[str],
     ):
+        if metadata.mime_type != "":
+            labels["mt"].add(metadata.mime_type)
```

Comment:
@bloodbare we already have a label for the icon `/n/i`. Shouldn't we reuse that one?

Reply:
/n/i is the icon of the resource where it belongs, not the mime tyupe of the field